### PR TITLE
Add mime-types 3.2.2

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: packages/tfm-ror52-rubygem-mime-types/tfm-ror52-rubygem-mime-types.spec [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+
+%files
+%dir %{gem_instdir}
+%{gem_instdir}/Code-of-Conduct.md
```
Comment (ekohl) [inline packages/tfm-ror52-rubygem-mime-types/tfm-ror52-rubygem-mime-types.spec:83]: This is probably also a `%doc`.

**Comment by ehelms** [comment]
[test rpm]

**Comment by ehelms** [comment]
[test rpm]

**Comment by ehelms** [comment]
[test rpm]

**Pull request merged**